shipyard-controller/handler/projecthandler.go

```diff
@@ -280,6 +293,21 @@ func (ph *ProjectHandler) CreateProject(c *gin.Context) {
 		SetBadRequestErrorResponse(c, fmt.Sprintf(InvalidRequestFormatMsg, err.Error()))
 		return
 	}
+
+	automaticProvisioningURL := os.Getenv("AUTOMATIC_PROVISIONING_URL")
```

Comment:
Please handle new environment variables via EnvConfig. It is already used in project handler